CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -0,0 +1,99 @@
+# Course Development Week 5
+Course Development Week 5 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Dec/07/2022
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: public engagement: story-telling and presentation; activism and trust in scientists
+After this week, one understands the role, responsibilities and interaction of scientists with scientific communities and the general public. 
+Students have to give a presentation about their other assignments using skills learned in the course. 
+Further, the studends are able to formulate their opinion in a discussion on the role a scientist must take in a public debate.
+
+## Keywords student familiarity check week 5
+Some of the keywords in syllabs.MD are supposed to be new, while some are most probably familiar for students.
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|How storytelling works|Metaphors|
+|Science communication and journalism|Communicating uncertainty|
+|Activism||
+|trust in science||
+|Bounds of expertise||
+|Academic social contract||
+|Narratives and framing||
+|Difficult to reach audience||
+|Process and outcome||
+
+
+## Lecture: Week 5
+
+**Preparation:** Read article about how physics is represented in Dutch newspapers. https://jcom.sissa.it/archive/20/07/JCOM_2007_2021_A02
+
+**Part 1** A short “classic” lecture of 25 min on:\
+Academic social contract\
+Bounds of expertise\
+Naratives and framing\
+trust in science\
+Difficulties to reach audience\
+process and outcome\
+trust in science\
+
+**Part 2** Guest lecture & Discussion (60 min)\
+On representation of physics in the Netherlands\
+Lecture by e.g. Julia Cramer (Leiden), followed up by a plenary discussion.
+
+
+## Tutorial Week 5: What is the role of a scientist in society?\
+(2x 60 min)
+\
+**Topics in tutorial:**\
+Metaphors | Communicating uncertainty | possible roles of a scientist
+
+### Tutorial format
++ **Assignment objective**
+Students practice presenting and express their points of view, by reflecting on their role as a future scientist. They will use metaphors to communicate topics of their bachelor research. 
```

Comment:
But not every student will be a 'future scientist'; and we certainly don't want to convey that expectation! Perhaps add 'or citizen'?

Reply:
```suggestion
Students practice presenting and express their points of view, by reflecting on their role as a future scientist or citizen. They will use metaphors to communicate topics of their bachelor research. 
```